pyproject.toml

```diff
@@ -36,6 +36,7 @@ dependencies = [
   "pydantic_numpy==5.0.2",
   "gradio",
   "gradio_client",
+  "importlib_resources"
```

Comment:
lets not depend on a new library and use importlib.resources instead please. Otherwise looks good.

Reply:
I have fixed it